packages/core/phase1/triggers/TRPS0008.ts

```diff
@@ -12,17 +13,19 @@ const generator: TriggerGenerator = (hearingOutcome, options) => {
   }
 
   const offences = hearingOutcome.AnnotatedHearingOutcome.HearingOutcome.Case.HearingDefendant.Offence
-  for (const offence of offences) {
+
+  return offences.reduce((triggers: Trigger[], offence) => {
     const offenceCode = offence ? getOffenceCode(offence) : undefined
-    const results = offence.Result
-    for (const result of results) {
-      if (result.CJSresultCode === triggerResultCode || offenceCode === triggerResultCode.toString()) {
-        return [{ code: triggerCode, offenceSequenceNumber: offence.CourtOffenceSequenceNumber }]
-      }
+    const shouldGenerateTrigger = offence.Result.some(
+      (result) => result.CJSresultCode === triggerResultCode || offenceCode === triggerResultCode.toString()
```

Comment:
```suggestion
      (result) => result.CJSresultCode === triggerResultCode
```


I think we can remove the check for `offenceCode`. In our standing data we do not have offence code `3105`, so `offenceCode === triggerResultCode.toString()` always evaluates to false.
https://raw.githubusercontent.com/ministryofjustice/bichard7-next-data/6949fefdcbe79d5b8dd789f185abb72c0fcdfce4/output-data/data/offence-code.json